pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -230,6 +230,9 @@ type LogCollectorSpec struct {
 	// Periodicity is the periodicity of the log rotation
 	// +optional
 	Periodicity string `json:"periodicity,omitempty"`
+	// MaxLogSizeMB is the maximum size of the log in GiB
```

Comment:
keeping this `// MaxLogSize is the maximum size of the log per ceph daemons. For example: 500M, 1G` it will be easier in the script to get M instead of `Mi`